brave/brave-core
# show eTLD+1 prominently in request panel

Resolves https://github.com/brave/brave-browser/issues/21787

Request panel from dApp now will show full origin with eTLD+1 emphasized

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to https://app.uniswap.org/#/swap and click `Connect Wallet`
2. The `eTLD+1` part of the `URL` should be `Bold` in the `Connect with site` panel

![Screen Shot 2022-03-23 at 11 18 51 AM](https://user-images.githubusercontent.com/40611140/159806388-9b956429-0294-4f09-be66-7459dc9ed98d.png)

1. Go to https://app.uniswap.org/#/swap and choose a token to swap, click `Allow spend`
2. The `eTLD+1` part of the `URL` should be `Bold` in the `Allow Spend` panel

![Screen Shot 2022-03-23 at 11 15 09 AM](https://user-images.githubusercontent.com/40611140/159806389-1eb3ec97-bfa6-4f88-a833-f4dbad0ece52.png)

1. Go to chainlist.org and connect your wallet, choose a `Network` to add.
2. The `eTLD+1` part of the `URL` should be `Bold` in the `Allow Add Network` panel

![Screen Shot 2022-03-23 at 11 06 54 AM](https://user-images.githubusercontent.com/40611140/159806395-8db1b6d8-0285-4515-952a-9d8522bf45f0.png)

3. You should see another prompt to switch networks
4. The `eTLD+1` part of the `URL` should be `Bold` in the `Allow Switch Network` panel

![Screen Shot 2022-03-23 at 11 07 06 AM](https://user-images.githubusercontent.com/40611140/159806392-8c4c807d-92f4-46e4-a2e2-52e1e3937811.png)

5. Open the `Wallet Panel` and click on the `Connected` button to get the `Site Permissions` panel
4. The `eTLD+1` part of the `URL` should be `Bold` in the `Site Permissions` panel

![Screen Shot 2022-03-23 at 11 07 22 AM](https://user-images.githubusercontent.com/40611140/159806391-a6ddba3a-1b50-4204-af7d-63aa7305392d.png)

1. Go to app.skiff.org and connect your wallet
2. You will then be prompt to `Sign`, sign the the transaction
3. You will then be prompt to provide your public encryption key.
4. The `eTLD+1` part of the `URL` should be `Bold` in the `Public Encryption Key` panel

![Screen Shot 2022-03-23 at 1 26 34 PM](https://user-images.githubusercontent.com/40611140/159808072-d44b93ef-b18c-4969-9ee9-4c42b20b35f1.png)

1. Go to http://www.reddit.com______________________________________________.________________________._________ethereum.defiapplication.wormholenetwork.com______________________._.______________wwwwwwwwwwwwwwwwwwwwwwwwwwwwwww.xdv.cz.srv21.endora.cz/metapoc/index.html and connect your wallet
2. The `URL` should be wrapped and the `eTLD+1` part of the `URL` should be `Bold` in the `Connect with site` panel

![Screen Shot 2022-03-23 at 4 43 09 PM](https://user-images.githubusercontent.com/40611140/159808433-549e4407-69dd-436e-b9e6-d79788ea12f1.png)

3. Open the `Wallet Panel` and click on the `Connected` button to get the `Site Permissions` panel
4. The `URL` should be wrapped and the `eTLD+1` part of the `URL` should be `Bold` in the `Site Permissions` panel

![Screen Shot 2022-03-23 at 4 42 47 PM](https://user-images.githubusercontent.com/40611140/159808584-5a6a2992-0b61-463e-bfe6-89af374b7658.png)
